src/theme/TOC/styles.module.css

```diff
@@ -0,0 +1,16 @@
+.tableOfContents {
```

Comment:
Would convert this to tailwind as well to be consistent in the app moving forward.